gen/dcos-config.yaml

```diff
@@ -826,7 +826,7 @@ package:
       "oauth":{"enabled":false},
 {% endswitch %}
 {% endswitch %}
-      "tracking":{"enabled":{{ telemetry_enabled }},"metadata":{{ ui_telemetry_metadata }}}}}}
```

Comment:
Will the value of `mesos_container_log_sink`  already be double quoted?
If not we will have to wrap it like this `"{{ mesos_container_log_sink }}"`